src/NuGet.Clients/NuGetConsole.Host.PowerShell/PowerShellHost.cs

```diff
@@ -528,6 +548,16 @@ public bool Execute(IConsole console, string command, params object[] inputs)
                 throw new ArgumentNullException(nameof(command));
             }
 
+            // Command can come here from both PMC and PM UI.
+            if (console is IWpfConsole)
+            {
+                _pmcExecutedCount++;
+            }
+            else
+            {
+                _nonPmcExecutedCount++;
```

Comment:
Does this happen when an install script runs?

Reply:
Yes, it does. 
There could be other cases I am not aware of.